### PR TITLE
CRUNCH UI Touch Ups

### PR DESCRIPTION
**Pull request by kennyqiyenkan**
Changes Made:
- Removed the optional tabs that lets you switch categories
- Added status pill to main feature flag
- Status pills will now change when there are changes to userCapabilityJunctionDAO

### CODE REVIEW

File: src/foam/u2/crunch/CapabilityCardView.js [hunk 1]
```diff
@@ -101,5 +96,31 @@ foam.CLASS({
...
+
+  listeners: [
+    {
+      name: 'daoUpdate',
+      code: function() {
+        this.userCapabilityJunctionDAO.find(
+          this.AND(
+            this.OR(
+              this.AND(
+                this.NOT(this.INSTANCE_OF(this.AgentCapabilityJunction)),
```
Comment (kgrgreer) [inline src/foam/u2/crunch/CapabilityCardView.js]: Why is this needed?

File: src/foam/u2/crunch/CapabilityFeatureView.js [hunk 2]
```diff
@@ -8,28 +8,71 @@ foam.CLASS({
...
+    {
+      name: 'associatedEntity',
+      expression: function(data, subject) {
+        if ( ! data || ! subject ) return '';
+        return data.associatedEntity === foam.nanos.crunch.AssociatedEntity.USER ? subject.user : subject.realUser;
+      }
+    },
+    {
+      name: 'cjStatus',
+      factory: function() {
```
Comment (kgrgreer) [inline src/foam/u2/crunch/CapabilityFeatureView.js:58]: Add documentation:.

File: src/foam/u2/crunch/CapabilityFeatureView.js [hunk 3]
```diff
@@ -57,5 +106,31 @@ foam.CLASS({
...
+  listeners: [
+    {
+      name: 'daoUpdate',
+      code: function() {
+        this.userCapabilityJunctionDAO.find(
+          this.AND(
+            this.OR(
+              this.AND(
+                this.NOT(this.INSTANCE_OF(this.AgentCapabilityJunction)),
+                this.EQ(this.UserCapabilityJunction.SOURCE_ID, this.associatedEntity.id)
```
Comment (kgrgreer) [inline src/foam/u2/crunch/CapabilityFeatureView.js]: Couldn't this clause be moved to the top-level AND?

Comment (Nauna) [reply src/foam/u2/crunch/CapabilityFeatureView.js]: ya we can clean this a litte. Suggest:
```
this.userCapabilityJunctionDAO.find(
          this.AND(
	    this.EQ(this.UserCapabilityJunction.TARGET_ID, this.data.id),
            this.EQ(this.UserCapabilityJunction.SOURCE_ID, this.associatedEntity.id),
            this.OR(
              this.NOT(this.INSTANCE_OF(this.AgentCapabilityJunction)),
              this.AND(
                this.INSTANCE_OF(this.AgentCapabilityJunction),
                this.EQ(this.AgentCapabilityJunction.EFFECTIVE_USER, this.subject.user.id)
              )
            )
          )
```

Comment (kennyqiyenkan) [reply src/foam/u2/crunch/CapabilityFeatureView.js]: Using @Nauna predicate, the results are the same.
